docs/tasks/tools/install-kubectl.md

```diff
@@ -146,6 +146,11 @@ If you see a message similar to the following, kubectl is not correctly configur
 ```shell
 The connection to the server <server-name:port> was refused - did you specify the right host or port?
 ```
+If kubectl cluster-info returns the url response but you can't access your cluster then to check whether it is configured properly or not use:
```

Comment:
>cluster then to check whether it is configured properly or not use:

cluster, to check whether it is configured properly, use:

Reply:
Applied changes